README.md

```diff
@@ -1,2 +1,50 @@
-# Solana Improvement Documents
-Solana IMprovement Documents (SIMD) describe proposed and accepted changes to the Solana protocol.
+# Solana Improvement Documents (SIMDs)
+
+The goal of the SIMD project is to standardize and provide high-quality
+documentation for Solana and its ecosystem. This repository tracks past and
+ongoing improvements to Solana in the form of Solana Improvement Documents
+(SIMDs).
+[SIMD-0001](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0001-simd-process.md)
+governs the SIMD process.
+
+## SIMD Types
+
+SIMDs can be divided into the following categories:
+
+- **Standard SIMDs**:
+  Describe changes that affect most or all Solana implementations, such as:
+  - **Core**:
+    Changes affecting consensus or substantial changes to the validator.
+  - **Networking**:
+    Changes or substantial improvements to network protocol specifications.
+  - **Interfaces**:
+    Breaking changes around the client JSON RPC API specifications and standards.
+- **Meta SIMDs**:
+  Describe a process surrounding Solana or propose a change to (or an event in)
+  a process.
+
+## Before You Begin
+
+Before you write a SIMD, ideas MUST be thoroughly discussed and vetted on the
+[ideas section](https://github.com/solana-foundation/solana-improvement-documents/discussions/categories/ideas)
+within this
+[repo's disucssion page](https://github.com/solana-foundation/solana-improvement-documents/discussions).
+Read and review [SIMD-0001](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0001-simd-process.md),
+which describes the SIMD process in detail.
+
+This repository is for documenting standards and not for implementation help.
+For specific questions and concerns regarding SIMDs, it's best to discuss them
+in the [questions section](https://github.com/solana-foundation/solana-improvement-documents/discussions/categories/questions)
+of this [repo's disucssion page](https://github.com/solana-foundation/solana-improvement-documents/discussions).
+
+## Access Policy
+
+The SIMD repository has three levels of access, as detailed in
+[SIMD-0007](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0007-access-policy.md):
+
+1. Triage
+2. Write
+3. Maintain
+
+To request access or report misuse, please follow the procedures outlined in
+SIMD-0007.
```

Comment:
it will probably pay to avoid info duplication here and simply link out to simd-0007 with a bit of guiding commentary

Reply:
I went back and forth on this my self. I'd like to keep it the same but only hold that opinion loosely